packages/twenty-front/src/pages/settings/developers/webhooks/SettingsDevelopersWebhookDetail.tsx

```diff
@@ -71,6 +84,78 @@ export const SettingsDevelopersWebhooksDetail = () => {
     })),
   ];
 
+  useEffect(() => {
```

Comment:
we don't put useEffects in component as this is a symptom that the component needs to re-render.
Instead, could you introduce a SettingsDevelopersWebhookUsageGraph.tsx component (that will contain your graph), and a SettingsDevelopersWebhookUsageGraphEffect.tsx (that will contain this useEffect).
You will also need to introduce a recoilState to store your data and so that your Graph can communicate with your GraphEffect